css-view-transitions-1/Overview.bs

```diff
@@ -225,8 +225,8 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 		```
 
 		Note: This pseudo-element provides a containing block for all ''::view-transition-group()'' pseudo-elements.
-		The aim of the style is to size the pseudo-element to cover the [=large viewport size=]
-		and position all ''::view-transition-group()'' pseudo-elements relative to the origin of the large viewport.
+		The aim of the style is to size the pseudo-element to cover the [=snapshot viewport=]
```

Comment:
I just realized this PR missed the part about UA styles on this pseudo-element. `inset: 0` is no longer correct.

- The first part is updating the style block for html::view-transition to:
   ```css
      position: fixed;
      top: 0;
      left: 0;
   ```

- The second part is to add a point in Style transition pseudo-elements algorithm. Something like:
   1. Let |root| be the ''::view-transition'' pseudo-element.
      1. Set "width" and "height" to snapshot viewport bounds.
      2. Set "transform" to a CSS transform that would place element at snapshot viewport origin.